solutions.txt

```diff
@@ -52,10 +47,13 @@ Exercise 5:
 
 // Complete Exercise 5 below...
 
-foods.splice(2, 0, 'tofu');
-
-
+foods.pop();
+foods.push('tofu', 'cheeseburger');
 console.log('Exercise 5 Result:\n', foods);
+//I'm pretty sure there is a better way of doing this but this is only solution I 
+knew how to do. I did a bunch of googling
+ and research and even youtube but I found this solution made sense to me.
```

Comment:
this works fine! for now, don't worry about "the best way to do things" -- the best way to do things is the way you can make it work that makes sense to you! You will learn best practices as you go along, and the more you code the more you will remember fancy methods that do things more concisely. But for now, the important thing is that you can think through the problem and use the tools you DO remember to solve it. Which you did here! 🥇